dependabot.yml

```diff
@@ -0,0 +1,6 @@
+version: 2
+updates:
+  - package-ecosystem: "npm" 
+    directory: "/"
+    schedule:
+      interval: "weekly"
```

Comment:
Can you add `reviewers` section with `khala`, `BlakeHolifield` and `florkbr`.

I'd also add `allow` section in here with options as:
```yaml
  - dependency-name: "@redhat-cloud-services/frontend*"
  - dependency-name: "@openshift/*"
  - dependency-name: "@patternfly/*"
    dependency-type: direct
```

We are not interested in other updates - they could break our app and needs to be updated with caution.